pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -225,13 +239,54 @@ def counting_pokemon(self, response_dict, id_list):
                     pass
                 else:
                     pokemon = item['inventory_item_data']['pokemon_data']
-                    if pokemon.get('is_egg', False):
-                        continue
-                    id_list.append(pokemon['id'])
+                    callback(pokemon)
 
-        return id_list
 
-    def should_release_pokemon(self, pokemon_name, cp, iv, response_dict):
+    # Returns True if pokemon is better than other_pokemon
+    def _is_better(self, pokemon, other_pokemon):
+        return (pokemon['cp'] >= other_pokemon['cp'])
```

Comment:
It should be `pokemon['cp'] > other_pokemon['cp']` I think, because with `=` we will have situation like this:

> [2016-07-24 23:56:22] [x] Captured Zubat! [CP 52] [IV 0.24]
> [2016-07-24 23:56:22] [#] Owning weaker Zubat(52). Replacing it with 52!
> [2016-07-24 23:56:23] [#] Weaker Zubat(52) has been exchanged for candy!
> [2016-07-24 23:56:23] [x] Captured Zubat! [CP 52]